src/application/mod.rs

```diff
@@ -335,6 +338,11 @@ impl Application {
         WindowProxy::new(self.application_proxy(), window_id)
     }
 
+    /// Set a file drop handler.
+    pub fn set_file_drop_handler(&mut self, handler: FileDropHandler) {
```

Comment:
I guess we could remove it and replace it with a FileDropHandler. Since it's an Arc users can close it to use it multiple times for different webviews. However this conflicts with your type trait request.

Reply:
Personally in the app I want to make using Tauri I would fine a global webview handler useful.

Reply:
What I mean is this method is not required in `Application` and such logic should be handled by users themselves.
Passing two different `FileDropHandler` seems redundant as a library.